### ISSUE TITLE
[reddit] always does not look like a valid subreddit.

### ISSUE THREAD

**Issue by jynxdarkspell** [TrustyJAID/Trusty-cogs#283]
No matter how I format the subreddit string so far, it always returns with 'whateverItype does not look like a valid subreddit."

**To Reproduce**
I've used:
([p] is substituted for prefix and whateveritype is substituted for the actual reddit name in these examples)
[p]redditset post whateveritype
[p]redditset post r/whateveritype
[p]redditset post /r/whateveritype
[p]redditset post /r/whateveritype/
[p]redditset post https://www/reddit.com//r/whateveritype
[p]redditset post https://www/reddit.com//r/whateveritype/

**Expected behavior**
What I expected to happen with one of these examples was to receive a message that the specified subreddit would now be posted in the channel the command was executed (it looks like I could execute the command and specify a discord channel, but it doesn't look like it's required)

**Additional context**
I've looked into the code, and I can see that including the base url is not necessary for me to include because the script should already be doing so, but I am unsure of the proper format for the subreddit string to make this cog function and there isn't an example in the help files or hidden in the code's comments.  I would love to be able to use this cog, as your cogs are "Trusty", as advertised!

**Comment by TrustyJAID** [comment]
Please ensure you follow the instructions in `[p]redditset creds`. You are required to provide your own reddit API tokens in order to access anything on reddit's API. There is also an issue if you try to use the reddit API with an account that has 2 factor authentication (2FA) enabled. I recommend creating a new reddit account without 2FA enabled and creating some API keys on that account to use the cog.

**Comment by jynxdarkspell** [comment]
I did follow the installation instructions, installing reactivepy,and git+https://github.com/TrustyJAID/aPRAW@setup-fix with the latest version of the red discord bot in a python 3.9 virtualenv.  I also followed the setup instructions, creating a new profile for the bot without 2FA, and setup my credentials as I was instructed.  I've even restarted the bot afterwards.  I've also re-verified that my credentials are correct and restarted my bot again.  I am still getting the same error no matter how I attempt to add the subreddit to my channel.
